docs/administration/configuration/configuring-external-access/index.md

```diff
@@ -0,0 +1,136 @@
+  extra:
+    - name: http
+      annotations:
+        kubernetes.io/ingress.class: traefik
+        traefik.ingress.kubernetes.io/router.entrypoints: http
+        traefik.ingress.kubernetes.io/router.middlewares: kube-system-redirect-scheme@kubernetescrd
+```
+
+## Manual setup
+
+If you would like to manually manage Ingresses, you should set `ingress.enabled` to false to disable the automatic creation of Ingresses.
+In the same way, you can disable the internal Traefik instance by setting `traefik.enabled` to false.
+
+Your custom Ingress must redirect all the requests to `/api` and `/oauth` to the `<helm-release-prefix>-server` Service, while all the rest should be sent to `<helm-release-prefix>-ui`
+You can find an example of custom Ingress below.
+
+??? note "Example of custom Ingress"
+
+    ```{ .yaml .copy-to-clipboard }
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
+    metadata:
+      name: reana-ingress
+      annotations:
+        kubernetes.io/ingress.class: traefik
+        traefik.ingress.kubernetes.io/router.entrypoints: web,websecure
+    spec:
+      tls:
+        - secretName: <tls-secret>
+      rules:
+        - http:
+            paths:
+            - path: /api
+              pathType: Prefix
+              backend:
+                service:
+                  name: <helm-release-prefix>-server
+                  port:
+                    number: 80
+            - path: /oauth
+              pathType: Prefix
+              backend:
+                service:
+                  name: <helm-release-prefix>-server
+                  port:
+                    number: 80
+            - path: /
+              pathType: Prefix
+              backend:
+                service:
+                  name: <helm-release-prefix>-ui
+                  port:
+                    number: 80
+          host: <hostname>
+    ```
+
+## Reverse proxy
+
+You might be interested in placing a (reverse) proxy in front of REANA, but some additional configuration might be needed to make REANA work correctly.
+
+Let's consider a setup in which a generic proxy handles connections to REANA, like so:
+
+```text
+          +---------+    +---------+
+Client -> |  Proxy  | -> |  REANA  |
+          +---------+    +---------+
+```
+
+In this case, REANA has no way to know how the client connected to the server, as REANA will only see connections coming from the proxy.
+If the proxy also performs TLS termination, all the connections made to REANA will be over HTTP, even if the client connected over HTTPS.
+This makes it impossible to construct correct URLs on the server-side, as the protocol to use is not known.
+
+Even though not standard, the `X-Forwarded-*` family of HTTP headers can be used to preserve the details of the original connection and make them known to REANA's web server.
+To fix the particular issue presented before, it is important that proxies correctly set the `X-Forwarded-Proto` HTTP header.
```

Comment:
It is  important to configure your proxy to provide correctly the `X-Forwarded-Proto` header.